mappings/net/minecraft/item/ArmorStandItem.mapping

```diff
@@ -2,5 +2,5 @@ CLASS azy net/minecraft/item/ArmorStandItem
 	METHOD <init> (Lbbw$a;)V
 		ARG 1 settings
 	METHOD a setRotations (Latd;Ljava/util/Random;)V
-		ARG 1 armorStand
+		ARG 1 entity
```

Comment:
Why would you change that ? The parameter is not any entity, it's an armor stand.

Reply:
Ah, this was from my initial pass at parameter mappings, cleaning up with generalized names. Do you think `stand` would suffice within this context?

Reply:
sure

Reply:
Resolved by 60cd923